serf/serf.go

```diff
@@ -1127,26 +1131,29 @@ func (s *Serf) handleNodeLeaveIntent(leaveMsg *messageLeave) bool {
 		}
 
 		if leaveMsg.Prune {
-			s.logger.Printf("[INFO] serf: EventMemberReap (forced): %s %s", member.Name, member.Member.Addr)
-			s.leftMembers = removeOldMember(s.leftMembers, member.Name)
-			s.eraseNode(member)
+			s.handlePrune(member)
 		}
 
 		return true
 
-	case StatusLeft:
```

Comment:
Did you intentionally remove handling for StatusLeft?

Reply:
Nope :(